src/common/SPELLS/paladin.ts

```diff
@@ -351,7 +391,12 @@ const spells = spellIndexableList({
     name: 'Zeal',
     icon: 'spell_holy_sealofblood',
   },
-  EXECUTION_SENTENCE_DEBUFF: {
+  //   EXECUTION_SENTENCE_DEBUFF: {
+  //     id: 343257,
+  //     name: 'Execution Sentence',
+  //     icon: 'spell_paladin_executionsentence',
+  //   },
```

Comment:
Remove this if it's unneeded or uncomment it

Reply:
Removed!